File: Serene/Serene.Script/Imports/ServiceContracts/Northwind.SalesByCategoryRow.cs

namespace Serene.Northwind
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Collections;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.Runtime.CompilerServices;

    [Imported, Serializable, PreserveMemberCase]
    public partial class SalesByCategoryRow
    {
        [InlineConstant] public const string NameProperty = "CategoryName";
        [InlineConstant] public const string LocalTextPrefix = "Northwind.SalesByCategory";
    
        public Int32? CategoryId { get; set; }
        public String CategoryName { get; set; }
        public String ProductName { get; set; }
        public Decimal? ProductSales { get; set; }
    
        [Imported, PreserveMemberCase]
        public static class Fields
        {
            [InlineConstant] public const string CategoryId = "CategoryId";
            [InlineConstant] public const string CategoryName = "CategoryName";
            [InlineConstant] public const string ProductName = "ProductName";
            [InlineConstant] public const string ProductSales = "ProductSales";
        }
    }
    
}

File: Serene/Serene.Core/Imports/ServerTypings/Northwind.SalesByCategoryRow.ts
namespace Serene.Northwind {
    export interface SalesByCategoryRow {
        CategoryId?: number;
        CategoryName?: string;
        ProductName?: string;
        ProductSales?: number;
    }

    export namespace SalesByCategoryRow {
        export const nameProperty = 'CategoryName';
        export const localTextPrefix = 'Northwind.SalesByCategory';

        export declare const enum Fields {
            CategoryId = "CategoryId",
            CategoryName = "CategoryName",
            ProductName = "ProductName",
            ProductSales = "ProductSales"
        }
    }
}
